Xamarin.Forms.Platform.WinRT/ListViewRenderer.cs

```diff
@@ -85,6 +85,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 				// WinRT throws an exception if you set ItemsSource directly to a CVS, so bind it.
 				List.DataContext = new CollectionViewSource { Source = Element.ItemsSource, IsSourceGrouped = Element.IsGroupingEnabled };
 
+				// This is a bit of a hack around the fact that SetupContent is not being called on the initial items
+				// in the ListView; iterating on each of the TemplatedItems gets around this as it then forces
+				// GetOrCreateContent to be called.
+				TemplatedItemsView.TemplatedItems.CollectionChanged += OnTemplatedItemsCollectionChanged;
```

Comment:
Is it possible to move this line before the setting of the DataContext above? I'm wondering if we can avoid having to duplicate the handler code on the next line.